crates/sdk/src/env.rs

```diff
@@ -109,9 +106,11 @@ pub fn read_register(register_id: RegisterId) -> Option<Vec<u8>> {
 #[inline]
 fn read_register_sized<const N: usize>(register_id: RegisterId) -> Option<[u8; N]> {
     let len = register_len(register_id)?;
-    let buffer = [0; N];
+
+    let mut buffer = [0; N];
+
```

Comment:
same here :)

Reply:
I don't personally like the space either. It's quite uncommon, and unidiomatic in Rust, to insert blank lines like this between assignments.

Reply:
Rust is generally kept compact, as the language is fairly terse, and brevity is sought-after. Plus, as the rustfmt standards lead to a lot of vertical code, it is undesirable to add more height.

I tend to add blank lines to differentiate between semantically-different blocks of code, which does aid clarity when there are a lot of lines. In short functions it provides no benefit and actually detracts.

Reply:
> It's quite uncommon and unidiomatic in Rust to insert blank lines like this between assignments.

There's no convention - https://github.com/rust-lang/style-team/issues/57

- https://github.com/rust-lang/rust/blob/28e8f01c2a2f33fb4214925a704e3223b372cad5/library/alloc/src/raw_vec.rs#L620
- https://github.com/sharkdp/bat/blob/eca6b8a3768ec3931a41330c42b138107b1786b5/src/diff.rs#L19

As you mentioned

> I tend to add blank lines to differentiate between semantically-different blocks of code, which does aid clarity when there are a lot of lines. In short functions it provides no benefit and actually detracts.

Below this, there's a macro annotation, merging them would impair clarity.

Reply:
I am not sure how it affects clarity, but even if wanting space between this assignment and the macro, I don't see why there's also a space between this assignment and the one above. It seems unnecessary.

I think there are more important things to worry about, but I tend to see these things as things I clean up from time to time in codebase hygiene sweeps. So I'm with @chefsale on this one.

Reply:
Shrunk